RFS/src/main/java/com/rfs/version_os_2_11/IndexCreator_OS_2_11.java

```diff
@@ -51,13 +51,13 @@ public Optional<ObjectNode> create(
 
             if (illegalArguments.isEmpty()) {
                 log.debug("Cannot retry invalid response, there are no illegal arguments to remove.");
-                return Optional.empty();
+                throw invalidResponse;
```

Comment:
Exceptions on index creation failures would have bubbled out before adding this retry logic.